android/app/src/googleplay/java/org/fossasia/openevent/fragments/MapsFragment.java

```diff
@@ -124,7 +118,29 @@ private void showLocationsOnMap() {
             builder.include(marker.getPosition());
         }
 
-        handleSearchEvents();
+        LayoutInflater inflator = (LayoutInflater) getActivity()
+                .getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+        View v = inflator.inflate(R.layout.map_toolbar, null);
+
+        toolbar.setCustomView(v);
+
+        //Setting up AutoCompleteTextView with the locations
+        ArrayAdapter<String> adapter = new ArrayAdapter<>(getActivity(), android.R.layout.simple_dropdown_item_1line, searchItems);
+        AutoCompleteTextView textView = (AutoCompleteTextView) v.findViewById(R.id.map_toolbar);
```

Comment:
Hence, this is the only reference view here, you can use `Butterknife.bind(this, v);` to correctly bind it.